src/factory/tests/set_token_bytecode.rs

```diff
@@ -0,0 +1,11 @@
+use factory::State;
```

Comment:
I wonder why put this test into a separate file, not as unit test inside the module? This is not an integration test, is it?